src/site/content/en/lighthouse-pwa/pwa-cross-browser/index.md

```diff
@@ -2,32 +2,41 @@
 layout: post
 title: Site works cross-browser
 description: |
-  Learn about `pwa-cross-browser` audit.
-date: 2019-05-02
+  Learn how to use Workbox to make sure your web page works across browsers.
```

Comment:
This doc focuses on Workbox a lot more than it should but I filed #1602 for us to figure this out at a later date. It's a manual audit so it's low priority.